pipeline-braille-scripts/epub3-to-pef/src/main/resources/xml/xproc/epub3-to-pef.convert.xpl

```diff
@@ -0,0 +1,300 @@
+    
+    <p:input port="fileset.in" primary="true"/>
+    <p:input port="in-memory.in" sequence="true"/>
+    <p:output port="fileset.out" primary="true">
+        <p:pipe port="result" step="fileset.out"/>
+    </p:output>
+    <p:output port="in-memory.out" sequence="true">
+        <p:pipe port="result" step="in-memory.out"/>
+    </p:output>
+    
+    <p:input kind="parameter" port="parameters" sequence="true">
+        <p:inline>
+            <c:param-set/>
+        </p:inline>
+    </p:input>
+    
+    <!-- Empty temporary directory dedicated to this conversion -->
+    <p:option name="temp-dir" required="true"/>
+
+    <p:import href="http://www.daisy.org/pipeline/modules/common-utils/library.xpl"/>
+    <p:import href="http://www.daisy.org/pipeline/modules/braille/common-utils/library.xpl"/>
+    <p:import href="http://www.daisy.org/pipeline/modules/braille/css-utils/library.xpl"/>
+    <p:import href="http://www.daisy.org/pipeline/modules/braille/pef-utils/library.xpl"/>
+    <p:import href="http://www.daisy.org/pipeline/modules/file-utils/library.xpl"/>
+    <p:import href="http://www.daisy.org/pipeline/modules/fileset-utils/library.xpl"/>
+    <p:import href="http://www.daisy.org/pipeline/modules/braille/xml-to-pef/library.xpl"/>
+    
+    <!-- Ensure that there's exactly one c:param-set -->
+    <p:identity>
+        <p:input port="source">
+            <p:pipe step="main" port="parameters"/>
+        </p:input>
+    </p:identity>
+    <p:wrap-sequence wrapper="c:param-set"/>
+    <p:unwrap match="/c:param-set/c:param-set"/>
+    <p:delete match="/c:param-set/c:param[@name = following-sibling::c:param/@name]"/>
+    <p:identity name="parameters"/>
+    <p:sink/>
+    
+    <!-- Load OPF and add content files to fileset. -->
+    <px:fileset-load media-types="application/oebps-package+xml">
+        <p:input port="fileset">
+            <p:pipe port="fileset.in" step="main"/>
+        </p:input>
+        <p:input port="in-memory">
+            <p:pipe port="in-memory.in" step="main"/>
+        </p:input>
+    </px:fileset-load>
+    <p:identity name="opf"/>
+    <p:xslt>
+        <p:input port="parameters">
+            <p:empty/>
+        </p:input>
+        <p:input port="stylesheet">
+            <p:document href="../xslt/opf-manifest-to-fileset.xsl"/>
+        </p:input>
+    </p:xslt>
+    <p:identity name="opf-fileset"/>
+    <p:sink/>
+    
+    <px:fileset-create name="temp-dir">
+        <p:with-option name="base" select="$temp-dir">
+            <p:empty/>
+        </p:with-option>
+    </px:fileset-create>
+    
+    <p:choose>
+        <p:xpath-context>
+            <p:pipe step="parameters" port="result"/>
+        </p:xpath-context>
+        <p:when test="not(/*/*[@name='toc-depth']/@value = '0')">
+            <px:fileset-add-tempfile media-type="text/css" suffix=".css">
+                <p:input port="source">
+                    <p:inline>
+                        <c:data>#generated-toc {
```

Comment:
In dtbook-to-pef I had moved this to the `default.scss` file (see https://github.com/daisy/pipeline-mod-braille/commit/090b1acdc235078de7d02e642bb9fa81c29205a0). I guess you didn't notice when you rebased this branch so now we have the rule defined twice.